OMNA/bcast/bcast-indentification-authority-registry.html

```diff
@@ -50,7 +50,7 @@
         <li>1.<i> {IAiD}</i> patterns starting with 'x-' are for testing and need not be coordinated with OMNA as they will not be recorded </li>
```

Comment:
This text is currently appearing in the original code. 
I'll review has this can be updated